pages/contribute/styleguide.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: page
+title: Style Guide
+search_exclude: false
```

Comment:
```suggestion
search_exclude: true
```
suggest to exclude contribute pages from search to not dilute the search on content